src/pages/privacy.tsx

```diff
@@ -1,1004 +1,30 @@
-import type { MouseEventHandler, ReactNode } from 'react'
+import MUILink from '@mui/material/Link'
+import type { LinkProps as MUILinkProps } from '@mui/material/Link/Link'
+import type { MDXComponents } from 'mdx/types'
+import type { LinkProps as NextLinkProps } from 'next/dist/client/link'
+import NextLink from 'next/link'
 import type { NextPage } from 'next'
 import Head from 'next/head'
 import { IS_OFFICIAL_HOST } from '@/config/constants'
+import SafePrivacyPolicy from '@/markdown/privacy/privacy.md'
 
-const SmoothScroll = ({ children }: { children: ReactNode }) => {
-  const onClick: MouseEventHandler = (e) => {
-    const anchor = (e.target as HTMLAnchorElement).getAttribute('href')
-    if (anchor?.startsWith('#')) {
-      e.preventDefault()
-      const element = document.querySelector(anchor)
-      if (element) {
-        element.scrollIntoView({ behavior: 'smooth' })
-      }
-    }
-  }
-
-  return <div onClick={onClick}>{children}</div>
+const CustomLink: React.FC<
+  React.PropsWithChildren<Omit<MUILinkProps, 'href'> & Pick<NextLinkProps, 'href' | 'as'>>
+> = ({ href = '', as, children, ...other }) => {
+  const isExternal = href.toString().startsWith('http')
+  return (
+    <NextLink href={href} as={as} passHref legacyBehavior>
+      <MUILink target={isExternal ? '_blank' : ''} rel="noreferrer" {...other}>
+        {children}
+      </MUILink>
+    </NextLink>
+  )
```

Comment:
How about we move this to util function as we have the same function in the terms.tsx page?

Reply:
done